txscript/standard.go

```diff
@@ -274,6 +289,7 @@ func CalcScriptInfo(sigScript, pkScript []byte, bip16 bool) (*ScriptInfo, error)
 			si.ExpectedInputs += shInputs
 		}
 		si.SigOps = getSigOpCount(shPops, true)
+		// TODO(roasbeef): segwit sig op counting
```

Comment:
Actually forgot to address before. The latest fix up includes a new set of tests for the possible combinations of `ScriptInfo` introduced as a result of segwit.